fw/Core/Hitcon/Logic/Display/display.h

```diff
@@ -176,5 +178,6 @@ class TextEditorDisplay;
 // Set the display to the text editor mode. The logic is outsourced to the
 // `TextEditorDisplay` class.
 void display_set_mode_editor(hitcon::TextEditorDisplay *editor);
+extern int display_set_mode_orientation;
```

Comment:
Would prefer making this part of DisplayService, but we probably don't have time.